### PR TITLE
Make paragraph line iteration scope

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
## Checklist

- [x] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [-] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [-] I have not broken the cheatsheet

### CODE REVIEW

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/modifiers/everyScope/clearEveryLine.yml
Comment (pokey) [inline packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/modifiers/everyScope/clearEveryLine.yml:1]: I would argue that both of these tests should be changed to "change every line file", and then add a new test for default iteration scope. Make sense?

**Review by pokey — COMMENTED** [review]
Looks good with minor comment

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/modifiers/everyScope/clearEveryLine.yml
Comment (AndreasArvidsson) [reply packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/modifiers/everyScope/clearEveryLine.yml:1]: done

File: packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/selectionTypes/takeEveryLine.yml [hunk 1]
```diff
@@ -18,8 +20,8 @@ initialState:
     d   e
     f   g
   selections:
-    - anchor: {line: 6, character: 0}
-      active: {line: 6, character: 0}
+    - anchor: {line: 5, character: 0}
+      active: {line: 5, character: 0}
```
Comment (pokey) [inline packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/selectionTypes/takeEveryLine.yml:24]: Any idea what's going on here?

Comment (AndreasArvidsson) [reply packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/selectionTypes/takeEveryLine.yml:24]: No when I test I get this new behavior for both the old implementation of take every line and the new one for take every line file. Looks like the test was just wrong somehow?

Comment (pokey) [reply packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/selectionTypes/takeEveryLine.yml:24]: Oh wait a minute. This is the initial state 😅

Comment (pokey) [inline packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/selectionTypes/takeEveryLine.yml:24]:
```suggestion
    - anchor: {line: 6, character: 0}
      active: {line: 6, character: 0}
```

**Review by pokey — APPROVED** [review]

**Pull request merged**